README.md

```diff
@@ -1,11 +1,48 @@
-# SYCL-Docs
-SYCL Open Source Specification
+![SYCL Logo](latex/logos/SYCL_RGB_June16-inkscape-1500.png)
+
+![Build Status](https://api.travis-ci.com/KhronosGroup/SYCL-Docs.svg?branch=master)
+[![SPEC master](https://img.shields.io/badge/SPEC-master-red.svg?logo=adobe-acrobat-reader)](https://khronosgroup.github.io/SYCL-Docs/sycl/sycl.pdf)
+[![SPEC 1.2.1-5](https://img.shields.io/badge/SPEC-1.2.1--5-orange.svg?logo=adobe-acrobat-reader)](https://www.khronos.org/registry/SYCL/specs/sycl-1.2.1.pdf)
+[![Join the Slack group](https://img.shields.io/badge/chat-on%20slack-blue.svg?logo=slack)](https://www.khronos.org/news/permalink/khronos-developer-slack-5bfc62eb261764.20435008)
+
+# SYCL Open Source Specification
 
 This repository contains the source and tool chain used to generate
-the formal SYCL specifications found on https://www.khronos.org/sycl/
+the formal SYCL specifications found on [https://www.khronos.org/sycl/](https://www.khronos.org/sycl/).
 
 ## Building
 
+For getting the dependencies the [Conan package manager](https://conan.io/) can be used. Building this way is more feature-complete, as it tracks all input files as dependencies of the `.pdf`, supports out-of-tree builds and downsamples/converts images for a smaller output.
+
+Packagers and distributors are advised to build the specification using this approach (or by calling CMake directly), since this will create an FHS-like directory structure with symlinks set up. Please refer to the `conanfile.py` for a full list of all dependencies.
+
+```bash
+# First time setup
+pip install --upgrade conan
+conan user
+conan remote add bincrafters https://api.bintray.com/conan/bincrafters/public-conan
+conan remote add mmha https://api.bintray.com/conan/mmha/public-conan
+
+# To actually build the spec:
+conan install . -if build --pr default --pr conan/profiles/sycl-spec-buildtools
+conan build . -bf build -pf package
```

Comment:
This particular error you got because you were somehow missing the default repository (conan-center), where CMake and ninja are located. Possibly because you `conan config install`ed a config without that remote. IIRC that was an earlier revision of this PR, sorry about that!

`conan remote add conan-center https://api.bintray.com/conan/conan/conan-center ` should fix it for you.

It's highly unlikely that a user misses `conan-center`. It's more likely for CI setups, but they'd `conan config install` this directory first. I'll add it to the README, regardless, since it doesn't hurt and there was at least one person with that problem :).